addon/components/frost-select.js

```diff
@@ -269,17 +283,37 @@ export default Ember.Component.extend({
     this.set('hovered', hovered)
   },
 
+  handleOutsideClick (event) {
```

Comment:
You can make this a const outside of the export/component definition if you want instead of the this.get() approach